src/backend/web/routes/login.js

```diff
@@ -21,13 +22,10 @@ router.use(passport.initialize({}));
 router.use(passport.session({}));
 
 /* this route is used to login and will redirect you to the external service that takes care of authentication */
-router.get('/login', handleLogin, passport.authenticate('samlStrategy'));
+router.get('/', handleLogin, passport.authenticate('samlStrategy'));
 
 /* this route is where you get directed from the external service. This is your redirect url found in the .env */
-router.post('/login/callback', handleLogin, passport.authenticate('samlStrategy'), function(
-  req,
-  res
-) {
+router.post('/callback', handleLogin, passport.authenticate('samlStrategy'), function(req, res) {
   logger.info({ user: req.user }, 'SSO login callback');
   res.send('Log in Callback Success');
```

Comment:
Necessary?  If needed, should this be text or JSON?  What does Seneca's SSO system do here, for comparison?

Reply:
I have it now returning a JSON object.